content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -60,6 +60,102 @@ for a number of reasons:
   without many constraints and have namespaced names, such configuration is
   usually portable.
 
+## Bound service account tokens
+
+ServiceAccount tokens can be bound to API objects that exist in the kube-apiserver.
+This can be used to tie the validity of a token to the existence of another API object.
+Supported object types are as follows:
+
+* Pod (used for projected volume mounts, see below)
+* Secret (can be used to allow revoking a token by deleting the Secret)
+* Node (beta in v1.30)
```

Comment:
One finicky part here - authenticating with node bound tokens is beta in 1.30, but actually issuing node bound tokens doesn't go beta until 1.31..